onomy/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "onomy",
+  "assets": [
+    {
+      "description": "The native token of Onomy Protocol",
+      "denom_units": [
+        {
+          "denom": "anom",
+          "exponent": 0
+        },
+        {
+          "denom": "nom",
+          "exponent": 18
+        }
+      ],
+      "base": "anom",
+      "name": "Nom",
+      "display": "nom",
+      "symbol": "NOM",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/onomy/images/nom.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/onomy/images/nom.svg"
+      },
+      "coingecko_id": "onomy-protocol",
```

Comment:
I don't see this in CoinGecko's list. https://api.coingecko.com/api/v3/coins/list
can we please remove until it's been added?

Reply:
Removed coingecko_id